docs/customizing-schemas/deprecating-schema.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: evolving-schema
-title: Evolving Schema
+id: deprecating-schema
```

Comment:
lets keep it as more generic evolving schema and add notes about adding new fields etc

Reply:
I was mainly doing this to help the sidebar be a little more clear. Right now it is not super descriptive for what is there.

Do you think we can just include this all on one page?


![Screen Shot 2020-02-03 at 2 48 46 PM](https://user-images.githubusercontent.com/2446877/73697815-601f7580-4694-11ea-9393-7047eb1b0030.png)

Reply:
The only info in the topic right now is on deprecating schema elements, so I'd recomment the title: Deprecating Schema Elements.

If or when you talk about other ways to evolve the schema or its elements, I'd change to something like Evolving Schema.

Just my two cents.